src/components/views/rooms/SendMessageComposer.js

```diff
@@ -0,0 +1,319 @@
+/*
+Copyright 2019 New Vector Ltd
```

Comment:
Is this file's content all new? Maybe C.I.C. only if so.

Reply:
There is code that was copied over from `MessageComposerInput.js`, so both is probably fine.